ui/app/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../tsconfig.base.json",
-  "include": ["src"],
+  "include": ["src", "../components/src/theme/palette/colors/types.ts", "../components/src/theme/palette/colors/grey.ts", "../components/src/theme/palette/secondary.ts", "../components/src/theme/palette/primary.ts"],
```

Comment:
This doesn't seem right. The `app` package shouldn't be building these files from the other package. Maybe this got updated automatically/accidentally when you moved the files?

Reply:
oh yep reverted this change!